dfinity/nns-dapp
# GIX-1890: Add Transaction Fee confirm screen

# PENDING

Confirm: The fee is not subtracted from the maturity. Instead, it just used as an arbitrary minimum to mint the tokens.

# Motivation

Add transaction fee in the confirmation screen when disbursing maturity.

# Changes

* Pass the `IcrcTokenMetadata` as prop instead of just the `tokenSymbol` in `DisburseMaturityModal`.
* Add one more KeyValuePair in `DisburseMaturityModal` confirmation screen.

# Tests

* Fill the tokensStore in spec file for SnsDisburseMaturityModal.
* Add expect in `"should display summary information in the last step"`.

# Todos

Not necessary. It's covered by the disburse maturity changelog entry.
